src/main/resources/jira/CheckIssue.py

```diff
@@ -31,8 +31,9 @@
         data = json.loads(response.getResponse())['fields']['status']
         issueStatus = data['name']
         print "\nIssue %s status is %s." % (issueId, issueStatus)
-        if issueStatus == expectedStatus:
-            print "\nThe status of issue %s is 'Resolved'" % (issueId)
+        if (issueStatus in expectedStatusList):
```

Comment:
Most styling guides would recommend removing the parentheses here - they are not needed.